timeless/templates/restaurants/tables/create_edit.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %}
+
+{% block header %}
+  <h1>{% block title %}Tables management - {% if action == 'create' %} Create {% else %} Edit {% endif %} {% endblock %}</h1>
+{% endblock %}
+
+{% block content %}
+  <form method="post">
+    <label for="name">Name</label>
+    <input name="name" id="name" required>
+    <br/>
+    <label for="floor_id">Floor ID</label>
+    <input name="floor_id" id="floor_id"/>
```

Comment:
@krzyk There should be a combo here showing name of the floor and the id as a value